content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx

```diff
@@ -29,7 +29,7 @@ Instead of `migrate deploy`, [`db push`](/concepts/components/prisma-migrate/db-
 
 When you add Prisma Migrate to an existing project, your initial migration contains all the SQL required to recreate the state of the database **before you started using Prisma Migrate**:
 
-![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases's features are represented by a single migration.](existing-database.png)
+![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases features are represented by a single migration.](existing-database.png)
```

Comment:
To indicate the posession still:
```suggestion
![The image shows a database labelled 'Existing database', and a list of existing database features next to it - 24 tables, 13 relationships, 92 fields, 3 indexes. An arrow labelled 'represented by' connects the database features list to a box that represents a migration. The existing databases' features are represented by a single migration.](existing-database.png)
```
Maybe?

Reply:
Thanks for your comments, removed this fix :)